demos/taco-demo/src/App.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  conditions,
+  decrypt,
+  encrypt,
+  getPorterUri,
+  initialize,
+  ThresholdMessageKit,
+} from '@nucypher/taco';
+import { Mumbai, useEthers } from '@usedapp/core';
+import { ethers } from 'ethers';
+import React, { useEffect, useState } from 'react';
+
+import { ConditionBuilder } from './ConditionBuilder';
+import { Decrypt } from './Decrypt';
+import { Encrypt } from './Encrypt';
+import { Spinner } from './Spinner';
+
+export default function App() {
+  const { activateBrowserWallet, deactivate, account, switchNetwork, chainId } =
+    useEthers();
+
+  const [loading, setLoading] = useState(false);
+  const [condition, setCondition] = useState<conditions.Condition>();
+  const [encryptedMessage, setEncryptedMessage] =
+    useState<ThresholdMessageKit>();
+  const [decryptedMessage, setDecryptedMessage] = useState<string>();
+  const [decryptionErrors, setDecryptionErrors] = useState<string[]>([]);
+
+  useEffect(() => {
+    initialize();
+    switchNetwork(Mumbai.chainId);
+  }, []);
+
+  const encryptMessage = async (message: string) => {
+    if (!condition) {
+      return;
+    }
+    setLoading(true);
+
+    await switchNetwork(Mumbai.chainId);
+
+    const provider = new ethers.providers.Web3Provider(window.ethereum);
+    const ritualId = 2; // Replace with your own ritual ID
```

Comment:
Is it possible to make this some sort of browser var or some variable that users can easily set somewhere - some kind of .env file equivalent for local scripts? Even if localized to a specific file.

Whatever process for updating this number should end up in the readme/demo instructions.